Libplanet/Store/StoreExtensions.cs

```diff
@@ -58,27 +58,5 @@ public static void Copy(this IStore from, IStore to)
                 to.SetCanonicalChainId(canonId);
             }
         }
-
```

Comment:
Can we delete `using Bencodex.Types` too?

Reply:
https://github.com/planetarium/libplanet/pull/1387/commits/7b2feba8d47ca2f6a5bd82c68abff466ef7f9eda